pkg/asset/machines/master.go

```diff
@@ -162,19 +162,41 @@ func (m *Master) Generate(dependencies asset.Parents) error {
 	machines := []machineapi.Machine{}
 	switch ic.Platform.Name() {
 	case alibabacloudtypes.Name:
+		client, err := installConfig.AlibabaCloud.Client()
```

Comment:
For future work, we should cache the VSwitches and availability zones so that we don't have to look them up multiple times.